lighthouse-core/gather/gatherers/full-page-screenshot.js

```diff
@@ -42,7 +42,7 @@ class FullPageScreenshot extends Gatherer {
     const height = Math.min(metrics.contentSize.height, MAX_SCREENSHOT_HEIGHT);
 
     await driver.sendCommand('Emulation.setDeviceMetricsOverride', {
-      mobile: passContext.baseArtifacts.TestedAsMobileDevice,
```

Comment:
It's a good point. This should be `screenEmulation.mobile`. In practice it doesnt matter because we already exited if there's a mismatch but I like keeping this correct!